testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/federation/ldap/AbstractLDAPTest.java

```diff
@@ -78,9 +80,11 @@ public void importTestRealms() {
         afterImportTestRealm();
     }
 
+    private static final String VAULT_EXPRESSION = "${vault.ldap_bindCredential}";
 
     protected void createLDAPProvider() {
         Map<String, String> cfg = getLDAPRule().getConfig();
+        cfg.put(BIND_CREDENTIAL, VAULT_EXPRESSION);
```

Comment:
This makes all LDAP tests to use vault secret. I would say we should remove it and use it only for some subset of LDAP tests to just confirm vault functionality. For these purposes, I created LDAPVaultCredentialsTest class.